src/components/atoms/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import HomeIcon from '@mui/icons-material/Home';
+import Link from '@mui/material/Link';
+import { Meta, StoryFn } from '@storybook/react';
+import { Box, Breadcrumbs, BreadcrumbsProps, Button } from 'src/components';
+
+export default {
+  title: 'Components/Breadcrumbs',
+  component: Breadcrumbs,
+} as Meta;
+
+const Template: StoryFn<BreadcrumbsProps> = (args) => <Breadcrumbs {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  children: [
+    <Button key="1" variant="text">
+      Menu Item
+    </Button>,
+    <Button key="2" variant="text">
+      Item
+    </Button>,
+    <Button key="3" variant="text">
+      Item 2
+    </Button>,
+  ],
+};
+
+export const WithIcons = Template.bind({});
+WithIcons.args = {
+  children: [
+    <Box key="box" display="flex" alignItems="center" gap={1}>
+      <HomeIcon fontSize="small" />
+      Menu Item
+    </Box>,
+    <Button key="7" variant="link">
+      Item
+    </Button>,
+    <Button key="8" variant="link">
+      Item 2
+    </Button>,
+  ],
+};
+
+export const WithButtons = Template.bind({});
```

Comment:
I don't see much usefulness in this variant. It's purpose I think is to show what we could render in the breadcrumb but I think that is achieved in all other variants and this in particular is a version we will never use. I suggest removing this.

Reply:
I would instead apply a variant with a custom selector